index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+<head>
+<link rel="stylesheet" href="style.css">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+<body>
+    <nav>
+        <ul>
```

Comment:
When should we use a list in code?
[Link ](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/li)to documentation on the list